test/e2e/network/service.go

```diff
@@ -1217,6 +1217,32 @@ var _ = SIGDescribe("Services", func() {
 		framework.ExpectNoError(err)
 	})
 
+	ginkgo.It("should be able to create a functioning ClusterIP service with ExternalIPs", func() {
+		serviceName := "externalip-test"
+		ns := f.Namespace.Name
+		externalIP := "203.0.113.250"
```

Comment:
add a comment explaining that these are IPs from the ranges reserved for documentation and examples, so we know they can't actually be in use on the host